Makefile

```diff
@@ -188,9 +190,11 @@ check: $(CARGO_OSDK)
 		(echo "Error: STD_CRATES and NOSTD_CRATES combined is not the same as all workspace members" && exit 1)
 	@rm /tmp/all_crates /tmp/combined_crates
 	@for dir in $(NON_OSDK_CRATES); do \
+		echo "Checking $$dir"; \
 		(cd $$dir && cargo clippy -- -D warnings) || exit 1; \
 	done
 	@for dir in $(OSDK_CRATES); do \
+		echo "Checking $$dir"; \
 		(cd $$dir && cargo osdk clippy -- -- -D warnings) || exit 1; \
```

Comment:
Em, yes. However forwarding commands like clippy for non-osdk crates have a little bit of difficulties since:
 - forwarded arguments are not parsed by osdk;
 - the definition is not clear: cargo won't check all but the default member unless `--workspace` is explicitly specified.
We need to redesign the CLI for the "running shortcuts in workspace" feature. Or I just want to leave as it is right now. QaQ.